design/secrets.md

```diff
@@ -92,15 +174,32 @@ Users will be able to set credentials during install and a secret will be create
 This secret will still be mounted into the Velero pods and the appropriate environment variables set.
 This will allow users to use versions of plugins which haven't yet been updated to use credentials directly, such as with many community created plugins.
 
-Multiple credential handling will only be used in the case where a particular BSL/VSL has been modified to use an existing secret.
+Multiple credential handling will only be used in the case where a particular BSL has been modified to use an existing secret.
 
 ## Security Considerations
 
 Although the handling of secrets will be similar to how credentials are currently managed within Velero, care must be taken to ensure that any new code does not leak the contents of secrets, for example, including them within logs.
 
+## Parallelism
+In order to support parallelism, Velero will need to be able to use multiple credentials simultaneously with the
+ObjectStore.  Currently backups are single threaded and a single BSL will be used throughout the entire backup.  The only
+existing points of parallelism are when a user downloads logs for a backup and when the .  In the current code, `download_request_controller.go`
```

Comment:
This sentence ends abruptly, should it be like "when a user downloads logs for a backup while a backup or restore is in progress"?

Reply:
Thanks for catching that.  I added the other case there and cleaned it up a bit.